internal/tiltfile2/k8s.go

```diff
@@ -132,10 +133,22 @@ func (s *tiltfileState) yamlEntitiesFromSkylarkValue(v skylark.Value) ([]k8s.K8s
 		if err != nil {
 			return nil, err
 		}
-		return k8s.ParseYAMLFromString(string(bs))
+		entities, err := k8s.ParseYAMLFromString(string(bs))
```

Comment:
would be slightly more idiomatic for the err != nil check to live here instead of the handle function